serverless-configs/aws-functions.yml

```diff
@@ -51,6 +51,21 @@
     timeout: 900
 
   #
+  # Generate Jupyter Notebook Lambda
+  #
+  generateNotebook:
+    handler: serverless/src/generateNotebook/handler.default
+    timeout: ${env:LAMBDA_TIMEOUT, '30'}
+    events:
+      - http:
+          method: post
+          cors: ${file(./serverless-configs/${self:provider.name}-cors-configuration.yml)}
+          path: generateNotebook
```

Comment:
Our lambdas actually use underscores instead of snake-case (not sure why). But lets do `generate_notebook` to be consistent with the other lambda paths